react/Price.tsx

```diff
@@ -53,7 +53,7 @@ const Price: StorefrontFunctionComponent<PriceProps> = ({
         id={`price-${item.id}`}
         className={`${handles.productPrice} div fw6 fw5-m`}
       >
-        {item.sellingPrice && (
+        {item.sellingPrice != null && (
```

Comment:
Does it come `null` from API or do we set this in somewhere else in our code?

Reply:
I honestly don't know. The issue here was that `0` is considered a "falsy" value (so it didn't render the `FormattedPrice`) but it is a value that React is able to render, so it rendered the zero instead of the "free" label

Reply:
The typings says it can be null because in our graphql we typed it as a [nullable value](https://github.com/vtex/checkout-graphql/blob/master/graphql/types/Item.graphql#L24). I think I had one situation where the price was null that was when the store owner didn't register the price on that specific item yet (the item has an availability of `withoutPriceFullfilment` or something)

Reply:
Considering the `tbd` scenario https://github.com/vtex-apps/formatted-price/blob/master/react/useFormattedPrice.ts#L21 (when it's `null`), shouldn't we check if it differs from `undefined`?

Reply:
the `sellingPrice != null` already covers both `null` and `undefined` and is the same check used in the formatted price app (notice the `==` versus `===`)

Reply:
But it means that we will never show the `tdb` message, right? Don't we want to allow it?

Reply:
hmmm good point, so you suggest to always render the `FormattedPrice` and let it handle the edge cases for the price?

Reply:
Yeap 😄 If something weird happen we should fix there (considering we want a standard solution for prices).

Reply:
:topzera:

Reply:
done in 5631db8